src/vtcomposite.cpp

```diff
@@ -771,15 +776,14 @@ struct LocalizeWorker : Napi::AsyncWorker
                                 {
                                     std::string property_value = static_cast<std::string>(property.value().string_value());
 
-                                    std::vector<std::string> available_worldviews = utils::split(property_value);
-
                                     // determine which worldviews to create a clone of the feature
                                     if (keep_every_worldview)
                                     {
-                                        worldviews_to_create = worldviews_for_feature(available_worldviews, available_worldviews);
+                                        worldviews_to_create = {property_value};
```

Comment:
property_value is converted to std::vector<std::string> with  utils::split previously. 
And worldviews_to_create is defined as  std::vector<std::string>

Would it be safer to keep the same data type as array like 
worldviews_to_create = available_worldviews;
so the line 774-775 still keep in the origin place?

Reply:
That'll require also changing https://github.com/mapbox/vtcomposite/blob/ad047c437b034a728b37a834ed7b5e503d48c26a/src/vtcomposite.cpp#L885 and will result in one copy of the feature for each string in the vector (i.e. larger file).